XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1372,10 +1368,6 @@
             {
               "Title": "Expander - API",
               "Uri": "https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.controls.expander"
-            },
-            {
-              "Title": "Guidelines",
-              "Uri": "https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/expander"
```

Comment:
@anawishnoff @kat-y @mikehillberg I wasn't able to find any "pattern" link for expander, is this intentional?